elastic-search-support/src/main/resources/application.yml

```diff
@@ -98,3 +98,6 @@ elasticsearch:
     '[@version]': ${elasticsearch.elasticMappings.disabled}
     index_id: ${elasticsearch.elasticMappings.disabled}
   securityClassificationMapping: ${elasticsearch.elasticMappings.defaultKeywordWithoutNormalizer}
+  eventCasePredefinedMappings:
+    eventId: ${elasticsearch.elasticMappings.defaultText}
+    case_type_id: ${elasticsearch.elasticMappings.defaultText}
```

Comment:
I added these 2 basic  mappings and the code is going to create complex types such us data